docs/kbs.yaml

```diff
@@ -71,19 +71,37 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorInformation'
 
-  /resources/key/<key_id>:
+  /resources/{repository}/{type}/{tag}:
     get:
-      operationId: getKey
-      summary: Get a key from the Key Broker Service.
+      operationId: getResource
+      summary: Get a secret resource from the Key Broker Service.
       parameters:
         - in: cookie
           name: kbs-session-id
           schema:
             type: string
+        - name: repository
+          in: path
+          description: A parent path of resource, can be empty to use the default repository.
```

Comment:
I am not sure what this is for. Could you give an example here?

Reply:
This is similar to the concept of container image repository, which is used to facilitate users to manage different resource groups. Its name should be completely set by users.

For example, if the user has two TEE Pod instances, the `repository` field can be used to manage the resources that need to be provided to the two TEE Pod instances:
```
/kbs/v0/resource/tee-instance-abcde123/key/<key-id>
/kbs/v0/resource/tee-instance-edcba321/key/<key-id>
```

Reply:
And, if KBS needs to support multiple tenants in the future, repository can also be the name of the tenant, for example:

```
/kbs/v0/resource/Alice/key/<key-id>
```